splunk_eventgen/lib/plugins/output/httpevent.py

```diff
@@ -171,15 +171,15 @@ def _sendHTTPEvents(self, payload):
         numberevents = len(payload)
         self.logger.debug("Sending %s events to splunk" % numberevents)
         for line in payload:
-            self.logger.debug("line: %s " % line)
```

Comment:
I am not sure why changing it back to `debugv` here works for historical reason? But changing this back to `debugv` has the right data volume generated for `multiprocess` mode.